src/test/java/SetTest.java

```diff
@@ -0,0 +1,58 @@
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("Set 의 크기를 확인하는 테스트")
+	public void sizeTest() {
+		// when
+		int size = numbers.size();
+
+		// then
+		assertThat(size).isEqualTo(3);
+	}
+
+	@ParameterizedTest
+	@ValueSource(ints = {1, 2, 3})
+	@DisplayName("Set 내 파라미터로 받은 값이 존재하는지 확인하는 테스트")
+	public void containsTestWithValueSource(int number) {
+		// when
+		boolean isContainNumber = numbers.contains(number);
+
+		// when
+		assertThat(isContainNumber).isTrue();
```

Comment:
```suggestion
       assertThat(numbers).contains(1, 2, 3);
```

- assertion을 활용하여 의도를 분명히 드러내는 건 어떨까요